aks-deploy/README.md

```diff
@@ -0,0 +1,41 @@
+ 
+# GitHub Action for deploying to Azure K8S Service
+
+
+## Usage
+
+```
+
+
+action "Deploy to Azure Kubernetes Service" {
+  uses = "actions/azure/aks@master"
+  secrets = ["DOCKER_PASSWORD", "KUBE_CONFIG_DATA"]
+  env = {
+    AKS_CLUSTER_NAME = "<AKS Cluster Name>"
+    DOCKER_USERNAME = "<Docker Registry username>"
+    DOCKER_REGISTRY_URL = "<Docker Registry URL>"
+    CONTAINER_IMAGE_NAME = "<Container Image Name>" 
+  }
+}
+
+```
+
+
+### Secrets
+
+- `DOCKER_PASSWORD` – **Optional** if public registry or if trust has been established with AKS. This is the password used to log in to your Docker registry. 
+- `KUBE_CONFIG_DATA` - **Optional**. "kubectl config" file content with credentials for Kubernetes to access the cluster. If we use this option, we wont need ["Azure Login"] action as a precursor to this one. 
+
+
+
+### Environment variables
```

Comment:
Suggest using table formats instead, would be easy to read.

Reply:
Yeah, that would be easier, but currently all GH Actions are using list format. will prefer to be in sync with them